packages/sign_in_with_apple/lib/src/authorization_request.dart

```diff
@@ -44,9 +44,9 @@ enum AppleIDAuthorizationScopes {
 class AppleIDAuthorizationRequest implements AuthorizationRequest {
   const AppleIDAuthorizationRequest({
     this.scopes = const [],
-    this.nonce,
-    this.state,
-  }) : assert(scopes != null);
+    required this.nonce,
+    required this.state,
```

Comment:
```suggestion
    this.state,
```

Reply:
Right, here we don't want to force the external callers to pass in `null` if they don't want it.

Changed in 18ea5d4a82a94698e57d6cd0bd2f1f6f76e85c4e